app/_includes/plugins/hub-examples/consumer_groups.html

```diff
@@ -68,7 +68,7 @@
     {% if hub_example.render_yaml? %}
     {% navtab Declarative (YAML) %}
 
-Add this section to your declarative configuration file:
+Add this section to your declarative configuration or decK file:
```

Comment:
Is this true? Consumer groups are different between declarative config + deck

Reply:
Hmm. What does it looks like with declarative config? I think all of these examples are more accurate written for decK.

I wonder if it would make more sense to just label them "yaml" like the UI does.

Reply:
Rolling this change back, will fix separately. There's a few other places in the docs where we conflate decK/declarative config, so I'd rather handle them together.

Reply:
I don't currently know how to set up a db-less instance (we don't document it with docker, but I'm sure there's a way 🤔) so I wasn't able to test originally. But I'll give it a shot and try to figure it out separately from this PR.

Reply:
```
curl -Ls https://get.konghq.com/quickstart | bash -s -- -D
curl -X POST http://localhost:8001/config -d "$(cat config.yaml)"
```

The `-D` flag in quickstart means DBless